source/templates/_main.slim

```diff
@@ -57,7 +57,8 @@ main[ng-controller="MainController as mainCtrl" ng-init="mainCtrl.init()"]
 		p == data[:strings][:main][:footer][:editor_open_source]
 		== svg("heart")
 		p == replaced_string(data[:strings][:main][:footer][:check_out_source_code], [data[:routes][:other][:bitrise_workflow_editor_source_code]])
-		a.version[href="#{ replaced_string(data[:routes][:other][:bitrise_workflow_editor_current_release], [ENV['RELEASE_VERSION']]) }" target="_blank" rel="noreferrer noopener"] v#{ ENV['RELEASE_VERSION'] }
+		- if mode == "cli"
+			a.version[href="#{ replaced_string(data[:routes][:other][:bitrise_workflow_editor_current_release], [ENV['version']]) }" target="_blank" rel="noreferrer noopener"] v#{ ENV['version'] }
```

Comment:
since every master commit will be a new version, we no longer use tags for website releases -> this is only used for binary ones